architecture/nvidia-gpu-architecture-overview.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: ASSEMBLY
+[id="nvidia-gpu-architecture-overview"]
+= NVIDIA GPU architecture overview
+include::_attributes/common-attributes.adoc[]
+:context: nvidia-gpu-architecture-overview
+
+toc::[]
+
+NVIDIA supports the use of graphics processing unit (GPU) resources on {product-title}. {product-title} is a security-focused and hardened Kubernetes platform developed and supported by Red Hat for deploying and managing Kubernetes clusters at scale. {product-title} includes enhancements to Kubernetes so that users can easily configure and use NVIDIA GPU resources to accelerate workloads.
+
+The NVIDIA GPU Operator leverages the Operator framework within {product-title} to manage the full lifecycle of NVIDIA software components required to run GPU-accelerated workloads.
+
+These components include the NVIDIA drivers (to enable CUDA), the Kubernetes device plugin for GPUs, the NVIDIA Container Toolkit, automatic node tagging using GPU feature discovery (GFD), DCGM-based monitoring, and others.
+
+[NOTE]
+====
+The NVIDIA GPU Operator is only supported by NVIDIA. For more information about obtaining support from NVIDIA, see link:https://access.redhat.com/solutions/5174941[Obtaining Support from NVIDIA].
+====
+
+include::modules/nvidia-gpu-prerequisites.adoc[leveloffset=+1]
+// New enablement modules
+include::modules/nvidia-gpu-enablement.adoc[leveloffset=+1]
+
+include::modules/nvidia-gpu-bare-metal.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/ai-enterprise/deployment-guide-openshift-on-bare-metal/0.1.0/on-bare-metal.html[Red Hat OpenShift on Bare Metal Stack]
+
+include::modules/nvidia-gpu-virtualization.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/openshift-virtualization.html[NVIDIA GPU Operator with OpenShift Virtualization]
+
+include::modules/nvidia-gpu-vsphere.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/openshift/nvaie-with-ocp.html#openshift-container-platform-on-vmware-vsphere-with-nvidia-vgpus[OpenShift Container Platform on VMware vSphere with NVIDIA vGPUs]
+
+include::modules/nvidia-gpu-kvm.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://computingforgeeks.com/how-to-deploy-openshift-container-platform-on-kvm/[How To Deploy OpenShift Container Platform 4.13 on KVM]
+
+include::modules/nvidia-gpu-csps.adoc[leveloffset=+2]
+[role="_additional-resources"]
+.Additional resources
+* link:https://docs.nvidia.com/ai-enterprise/deployment-guide-cloud/0.1.0/aws-redhat-openshift.html[Red Hat Openshift in the Cloud]
+* link:https://access.redhat.com/solutions/6177662[OpenShift 4 and GPU requirements in Google Cloud Platform (GCP)]
```

Comment:
This article is a **Solution in Progress**. Do we have approvals to post a link to a non-verified solution?